guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.guvnor.structure.repositories;
+
+public interface PullRequest {
+
```

Comment:
Just curious.. why is PullRequest not defining the Author? Sounds like a basic piece of information to have in a PR.